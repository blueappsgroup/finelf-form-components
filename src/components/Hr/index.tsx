import React from 'react'
import styled from 'styled-components'

type StyledProps = {
  theme: {
    hrColor: string
    hrSize: string
  }
}

// prettier-ignore
const HrLine = styled.hr<StyledProps>`
  width: 100%;
  margin: 25px auto;
  border-color: ${(props: StyledProps): string => props.theme.hrColor};
  border-width: ${(props: StyledProps): string => props.theme.hrSize};
`

const Hr: React.FC = () => <HrLine />

export default Hr
