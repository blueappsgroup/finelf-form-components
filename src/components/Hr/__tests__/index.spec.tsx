import { render } from '@testing-library/react'
import * as React from 'react'
import Hr from '../'

describe('<Hr />', () => {
  it('hr matches snapshot', () => {
    const documentBody = render(<Hr />)
    const { baseElement } = documentBody

    expect(baseElement).toMatchSnapshot()
  })
})
