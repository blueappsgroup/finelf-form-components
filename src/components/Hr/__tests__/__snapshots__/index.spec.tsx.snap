// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hr /> hr matches snapshot 1`] = `
<body>
  <div>
    <hr
      class="sc-bdnylx jPdwJh"
    />
  </div>
</body>
`;
